src/configs/chain_config.mainnet.json

```diff
@@ -18,7 +18,7 @@
   "votingPowerTokenUnit": "ncheq",
   "tokenUnits": {
     "ncheq": {
-      "display": "CHEQ",
+      "display": "cheq",
```

Comment:
For display, we always stylise it as CHEQ (all caps) like ATOM...so not sure if this is correct

Reply:
In BDv2, display is the exact literal used by CoinGecko, so this is the correct one as fixed in [DEV-1090].
By default, BDv2 formats that in uppercase.